data/i18n/hi/hi.toml

```
@@ -70,6 +73,9 @@ other = "क्या आप यह ढूँढ रहे थें:"
 [examples_heading]
 other = "उदाहरण"
 
+[feature_state]
+other = "फ़ीचर राज्य:"
```

```suggestion
other = "फ़ीचर स्थिति:"
```
The word राज्य translates to "state" in geography. (Example: "There are 50 states in the US"). स्थिति is the correct translation here.

---

Updated. Thanks.